integration/setup/setup.go

```diff
@@ -87,12 +87,38 @@ func Resources(g *framework.Guest, h *framework.Host) error {
 	var err error
 
 	{
-		err = h.InstallStableOperator("cert-operator", "certconfig", e2etemplates.CertOperatorChartValues)
+		var certOperatorValues string
+		{
+			c := chartvalues.CertOperatorConfig{
+				ClusterName: env.ClusterID(),
+				ClusterRole: chartvalues.CertOperatorClusterRole{
+					BindingName: key.ClusterRole("cert-operator"),
+					Name:        key.ClusterRole("cert-operator"),
+				},
+				ClusterRolePSP: chartvalues.CertOperatorClusterRole{
+					BindingName: key.ClusterRolePSP("cert-operator"),
+					Name:        key.ClusterRolePSP("cert-operator"),
+				},
+				CommonDomain: env.CommonDomain(),
+				PSP: chartvalues.CertOperatorPSP{
+					Name: key.PSPName("cert-operator"),
+				},
+				RegistryPullSecret: env.RegistryPullSecret(),
+				Vault: chartvalues.CertOperatorVault{
+					Token: env.VaultToken(),
+				},
+			}
+			certOperatorValues, err = chartvalues.NewCertOperator(c)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+		}
+		err = h.InstallStableOperator("cert-operator", "certconfig", certOperatorValues)
```

Comment:
Can't we directly use the `release` package here?

Reply:
I want to do that in a follow up step because I want to sort out the clashes of resources first.

Only `node-operator` is missing, after that I will gradually use the `release` package for all of this.